mage_ai/data_preparation/models/block.py

```diff
@@ -359,7 +374,10 @@ def __validate_execution(self, decorated_functions, input_vars):
 
             return block_function
 
-    def execute_block(self, custom_code=None, redirect_outputs=False):
+    def execute_block(self, custom_code=None, redirect_outputs=False, global_vars=None):
+        if global_vars is None:
+            x = {}
```

Comment:
what's x?

Reply:
Typo, meant to type CTRL+x to remove that piece of code. Removed now.